sdk/data/azcosmos/README.md

```diff
@@ -0,0 +1,157 @@
+
+### Create an Azure Cosmos account
+
+You can create an Azure Cosmos account using:
+
+* [Azure Portal](https://portal.azure.com).
+* [Azure CLI](https://docs.microsoft.com/cli/azure).
+* [Azure ARM](https://docs.microsoft.com/azure/cosmos-db/quick-create-template).
+
+#### Install the package
+
+* Install the Azure Cosmos DB SDK for Go with `go get`:
+
+  ```bash
+  go get -u github.com/Azure/azure-sdk-for-go/sdk/data/azcosmos
+  ```
+  
+#### Authenticate the client
+
+In order to interact with the Azure CosmosDB service you'll need to create an instance of the Cosmos Client class. To make this possible you will need an URL and key of the Azure CosmosDB service.
+
+#### Create CosmosClient
+
+```go
+endpoint := "someEndpoint"
+key := "someKey"
+cred, _ := NewSharedKeyCredential(key)
+client, err := NewCosmosClient(endpoint, cred, nil)
+handle(err)
+```
+
+## Examples
+
+The following section provides several code snippets covering some of the most common CosmosDB SQL API tasks, including:
+* [Create Cosmos Client](#create-cosmos-client "Create Cosmos Client")
+* [Create Database](#create-database "Create Database")
+* [Create Container](#create-container "Create Container")
+* [CRUD operation on Items](#crud-operation-on-items "CRUD operation on Items")
+
+### Create Cosmos Client
+
+```go
+const (
+    CosmosDbEndpoint = "someEndpoint"
+    CosmoDbKey = "someKey"
+)
+
+cred, _ := NewSharedKeyCredential(CosmosDbEndpoint)
+client, err := NewCosmosClient(CosmoDbKey, cred, &CosmosClientOptions{})
+handle(err)
+```
+
+### Create Database
+
+Using the client created in previous example, you can create a database like this:
+
+```go
+database := CosmosDatabaseProperties{Id: dbName}
+response, err := client.CreateDatabase(context, database, nil, nil)
+handle(err)
+```
+
+### Create Container
+
+Using the above created database for creating a container, like this:
+
+```go
+response, err := database.CreateContainer(context, properties, throughput, nil)
+handle(err)
+container := resp.ContainerProperties.Container
+```
+
+### CRUD operation on Items
+
+```go
+properties := CosmosContainerProperties{
+    Id: "aContainer",
+    PartitionKeyDefinition: PartitionKeyDefinition{
+        Paths: []string{"/id"},
+    },
+}
+
+item := map[string]string{
+    "id":    "1",
+    "value": "2",
+}
+
+// Create partition key
+container := resp.ContainerProperties.Container
+pk, err := NewPartitionKey("1")
+handle(err)
+
+// Create an item
+itemResponse, err := container.CreateItem(context, pk, item, nil)
+handle(err)
+
+// Read an item
+itemResponse, err = container.ReadItem(context, pk, "1", nil)
+handle(err)
+
+// Replace an item
+itemResponse, err = container.ReplaceItem(context, pk, "1", item, &CosmosItemRequestOptions{EnableContentResponseOnWrite: true})
+handle(err)
+
+// Delete an item
+itemResponse, err = container.DeleteItem(context, pk, "1", nil)
+handle(err)
+```
+
+For many more scenarios and examples see
```

Comment:
Might want to remove, we are not sure about this section, could not find any other README that points to Azure-Samples/azure-sdk-for-go-samples